Makefile

```diff
@@ -338,8 +338,8 @@ run-container:
 	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/go/src/github.com/project-zot/zot \
 		zot-build:latest
 
-.PHONY: binary-stacker
-binary-stacker: $(STACKER)
+.PHONY: oci-image
```

Comment:
This target can be run only on linux. In fact $(STACKER) depends on check-linux target.

> aiandu@aiandus-MacBook-Pro zot % git diff .
> diff --git a/Makefile b/Makefile
> index f2ef779..d108907 100644
> --- a/Makefile
> +++ b/Makefile
> @@ -421,7 +421,7 @@ fuzz-all:
>         bash test/scripts/fuzzAll.sh ${fuzztime}; \
>         rm -rf pkg/storage/testdata; \
>  
> -$(STACKER):
> +$(STACKER): check-linux
>         mkdir -p $(TOOLSDIR)/bin; \
>         curl -fsSL https://github.com/project-stacker/stacker/releases/latest/download/stacker -o $@; \
>         chmod +x $@

Reply:
Correct, `stacker` is currently supported only on host platform linux/amd64. Target will depend on the toolchain used inside stacker.

Reply:
updated the Makefile rule